src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString.split(';');
```

Comment:
Consider adding a check to ensure `sourceString` is indeed a string before proceeding with the split operation. This will prevent potential runtime errors if a non-string type is passed.